modules/system/checks.nix

```diff
@@ -27,7 +27,7 @@ let
         echo "Create a symlink to /var/run with:" >&2
         if test -e /etc/synthetic.conf; then
             echo >&2
-            echo "$ echo 'run\tprivate/var/run' | sudo tee -a /etc/synthetic.conf" >&2
+            echo "$ printf 'run\tprivate/var/run\n' | sudo tee -a /etc/synthetic.conf" >&2
             echo "$ /System/Library/Filesystems/apfs.fs/Contents/Resources/apfs.util -B" >&2
```

Comment:
This flag (`-B`) doesn't exist anymore in `apfs.util`, so I got confused as to what to do with this command, when the solution was to just ignore this command. Should this line be removed, or is this relevant on older versions of macOS?

Reply:
Yeah the flags changed at some point so the installer tries both, this message was added before the flag changed.